scripts/apine.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+#
+# MIT License
+# Copyright (c) INCF
+#
+# apine.py
+# Utility for interacting with Apine objects
+# Created 10/12/2017 initially by G. Kiar
+#
+# To install dependencies, in a fresh virtual environment run:
+#    pip install jsonschema
+#
+
+from argparse import ArgumentParser
+from collections import OrderedDict
+from copy import deepcopy
+import os.path as op
+from jsonschema import validate, ValidationError
+import json
+
+
+def validateApineObj(dataset_descriptor, apine=None):
+    try:
```

Comment:
quick argment description following numpy way ? 
http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

Reply:
Good call - did it for both files.